commands.js

```diff
@@ -1760,15 +1760,39 @@ exports.commands = {
 		if (isWin) {
 			command = path.normalize(__dirname + '/lib/winmodlog') + ' tail ' + lines + ' ' + filename;
 		} else {
-			command = 'tail -' + lines + ' ' + filename;
+			command = 'tail -' + lines + ' ' + filename + ' | tac';
```

Comment:
if somebody wants to show me how to do this (reverse tail output line order) on a windows PS server, be my guest n_n

Code for tail on windows PS: https://github.com/Zarel/Pokemon-Showdown/blob/master/lib/winmodlog.cmd#L15